src/content/docs/ko/guides/deploy/aws.mdx

```diff
@@ -0,0 +1,279 @@
+---
+title: Astro 사이트를 AWS에 배포
+description: AWS를 사용하여 Astro 사이트를 웹에 배포하는 방법.
+type: deploy
+i18nReady: true
+---
+
+[AWS](https://aws.amazon.com/)는 Astro 사이트를 배포하는 데 사용할 수 있는 모든 기능을 갖춘 웹 앱 호스팅 플랫폼입니다.
+
+프로젝트를 AWS에 배포하려면 [AWS 콘솔](https://aws.amazon.com/console/)을 사용해야 합니다. (이러한 작업의 대부분은 [AWS CLI](https://aws.amazon.com/cli/)를 사용하여 수행할 수도 있습니다.) 이 가이드는 가장 기본적인 방법부터 시작하여 사이트를 AWS에 배포하는 단계를 안내합니다. 그런 다음 비용 효율성과 성능을 향상시키기 위해 서비스를 추가하는 방법을 보여줍니다.
+
+## AWS Amplify
+
+AWS Amplify는 프런트엔드 웹 및 모바일 개발자가 AWS에서 풀 스택 애플리케이션을 빠르고 쉽게 구축할 수 있도록 특별히 제작된 도구 및 기능 세트입니다.
+
+1. 새로운 Amplify 호스팅 프로젝트를 생성합니다.
+2. 저장소를 Amplify에 연결합니다.
+3. 빌드 출력 디렉터리 `baseDirectory`를 `/dist`로 수정합니다.
+
+    ```yaml
+    version: 1
+    frontend:
+      phases:
+        preBuild:
+          # npm을 사용하지 않나요? `npm ci`를 `yarn install` 또는 `pnpm i`로 변경하세요.
+          commands:
+            - npm ci
+        build:
+          commands:
+            - npm run build
+      artifacts:
+        baseDirectory: /dist
+        files:
+          - '**/*'
+      cache:
+        paths:
+          - node_modules/**/*
+    ```
+
+`pnpm`을 사용하려면 `node_modules` 대신 pnpm 저장소 디렉터리를 캐시하기 위해 약간 다른 설정이 필요합니다. 다음은 권장되는 빌드 구성입니다.
+
+    ```yaml
+    version: 1
+    frontend:
+      phases:
+        preBuild:
+          commands:
+            - npm i -g pnpm
+            - pnpm config set store-dir .pnpm-store
+            - pnpm i
+        build:
+          commands:
+            - pnpm run build
+      artifacts:
+        baseDirectory: /dist
+        files:
+          - '**/*'
+      cache:
+        paths:
+          - .pnpm-store/**/*
+    ```
+
+Amplify는 커밋을 저장소에 푸시할 때 자동으로 웹 사이트를 배포하고 업데이트합니다.
+
+## S3 정적 웹사이트 호스팅
+
+S3는 모든 애플리케이션의 시작점입니다. 여기에는 프로젝트 파일과 기타 자산이 저장됩니다. S3에서는 파일 저장 및 요청 수에 따라 요금을 청구합니다. [AWS 설명서](https://aws.amazon.com/s3/)에서 S3에 대한 자세한 내용을 확인할 수 있습니다.
+
+1. 프로젝트 이름으로 S3 버킷을 생성합니다.
+
+    :::tip
+    버킷 이름은 전역적으로 고유해야 합니다. 프로젝트 이름과 사이트 도메인 이름의 조합을 권장합니다.
+    :::
+
+2. **"Block all public access"**를 비활성화합니다. 기본적으로 AWS는 모든 버킷을 비공개로 설정합니다. 공개하려면 버킷 속성에서 "Block public access" 확인란을 선택 취소해야 합니다.
```

Comment:
Bold here isn't working, probably adding a space would fix it?
```suggestion
2. **"Block all public access"** 를 비활성화합니다. 기본적으로 AWS는 모든 버킷을 비공개로 설정합니다. 공개하려면 버킷 속성에서 "Block public access" 확인란을 선택 취소해야 합니다.
```

Reply:
Thanks for the good point 👍